creator-node/src/snapbackSM/syncHistoryAggregator.js

```diff
@@ -74,10 +78,17 @@ class SyncHistoryAggregator {
       const latestSyncKeyTTL = await this.getKeyTTL(latestSyncKeys[state])
       await redisClient.set(latestSyncKeys[state], timeOfEvent, 'EX', latestSyncKeyTTL)
 
-      logger.info(`SyncHistoryAggregator - Successfully tracked ${state} sync at ${timeOfEvent}`)
+      // Update per wallet success/fail sync data
+      // Each daily wallet key stores a set of wallets, representing each unique user with a sync in that state that day
+      const dailyWalletSyncKey = SyncHistoryAggregator.getUniqueSyncKeys()
+      const dailyWalletSyncKeyTTL = await this.getKeyTTL(dailyWalletSyncKey[state])
```

Comment:
this is my fault, but we probably should do `SyncHistoryAggregator.getKeyTTL(...)` cus using `this` in a static file is strange even though it works

Reply:
could we change this across this file?

Reply:
done!

Reply:
interesting i've generally been moving in the opposite direction